jvav/utils.py

```diff
@@ -130,12 +130,43 @@ def write_html(resp: requests.Response):
 
 class JavDbUtil(BaseUtil):
     BASE_URL = "https://javdb.com"
-    BASE_URL_SEARCH = "https://javdb.com/search?q="
+    BASE_URL_SEARCH = BASE_URL + "/search?q="
+    BASE_URL_VIDEO = BASE_URL + "/v/"
+    BASE_URL_ACTOR = BASE_URL + "/actors/"
 
-    def get_ids_from_page(self, url: str) -> typing.Tuple[int, list]:
-        """从某个页面获取番号列表
+    def __init__(self, proxy_addr=""):
+        """
+        初始化
+        :param proxy_addr: 代理服务器地址，使用本机软件科学上网的话，格式通常为'http://127.0.0.1:{port}'
+        """
+        super().__init__(proxy_addr)
 
-        :param str url: 某个页面
+    # 转换ID为JavDB ID
+    def get_javdb_id_by_id(self, id: str) -> tuple[int, None] | tuple[int, typing.Any]:
```

Comment:
use typing.Tuple instead of tuple plz, some old version python3 does not support "tuple" keyword